### PR TITLE
Add Lippupiste as an external ticket system

### PR DESCRIPTION
**Pull request by nikomakela**
KK-987 KK-994.

The integration for the external ticket systems,
the Ticketmaster and the Lippupiste, should be similar.
Only thing that varies at this point , is the ticket system URL.

The old Ticketmaster model and schema classes are now renamed
and converted to some abstract classes. The old Ticketmaster class names
are now given to the new classes that inherits the abstracts.
New Lippupiste classes are copied from the new Ticketmaster ones.

The old implementation for Ticketmaster should be unchanged,
so all the existing unit tests should pass without any changes.
There are now some changes added there only to randomly test that also
the Lippupiste events are calculated in same manners.

### CODE REVIEW

**Review by karisal-anders — COMMENTED** [review]
Why the three added and commented out `get_queryset` functions?

**Comment by nikomakela** [comment]
**Let's wait for product owner approval before merging this!** 🚫

**Review by HeikkM — APPROVED** [review]

**Comment by terovirtanen** [comment]
**KUKKUU-API branch is deployed to platta: https://kukkuu-pr313.api.dev.hel.ninja** :rocket::rocket::rocket:

**Pull request merged**
